seacrowd/sea_datasets/newsph/newsph.py

```diff
@@ -0,0 +1,128 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{cruz2020investigating,
+  title={Investigating the True Performance of Transformers in Low-Resource Languages: A Case Study in Automatic Corpus Creation},
+  author={Jan Christian Blaise Cruz and Jose Kristian Resabal and James Lin and Dan John Velasco and Charibeth Cheng},
+  journal={arXiv preprint arXiv:2010.11574},
+  year={2020}
+}
+"""
+_DATASETNAME = "newsph"
+_LANGS = ["fil", "tgl"]
+_DESCRIPTION = """\
+Raw collection of news articles in Filipino which can be used for language modelling.
+"""
+_HOMEPAGE = "https://huggingface.co/datasets/newsph"
+_LICENSE = Licenses.GPL_3_0.value
+_LOCAL = False
+_URLS = "https://s3.us-east-2.amazonaws.com/blaisecruz.com/datasets/newsph/newsph-nli.zip"
```

Comment:
@IvanHalimP I think you used the wrong data URL. This dataloader is supposedly for [NewsPH](https://huggingface.co/datasets/newsph), which is a language modeling task, as provided in the #106. The URL that you used is supposedly for NewsPH-NLI, a derivative of NewsPH.

May I suggest you to:
1. Use the `LANGUAGE_MODELING` task and [this data URL](https://huggingface.co/datasets/newsph) for NewsPH dataloader.
2. Submit the NewsPH-NLI datasheet via [our form](https://form.jotform.com/team/232952680898069/seacrowd-sea-datasets).
3. Make a pull request using the current dataloader for NewsPH-NLI after the issue comes up (+ modify some of the constants, e.g., `_DESCRIPTION`, `_SUPPORTED_TASKS`, etc., accordingly)?

Please let me know what you think and update me on your next course of action so I can help.

Reply:
Hi all, thank you for the comments. 
Please kindly wait for the time being as I need to take care of something else.

Reply:
@IvanHalimP Sure sure, just let us know if we can help with anything. Enjoy your holiday~

Reply:
Regarding the related task for this newsph dataset, there is no labels and no pairs. AFAIK, it falls into self-supervised pretraining schema then?

Reply:
@IvanHalimP Yes yes. I saw your changes and tested them. The dataloader works well on my end. Thanks for the modifications. I'm merging it now.